app/carbonapi/app_test.go

```diff
@@ -322,3 +328,14 @@ func TestInfoHandler(t *testing.T) {
 		t.Error("Http response should be same.")
 	}
 }
+
+func setupTestServer(testHandler http.Handler) *httptest.Server {
+	testListener, err := net.Listen("tcp", "127.0.0.1:8080")
+	if err != nil {
+		log.Fatal(err)
+	}
+	ts := httptest.NewUnstartedServer(testHandler)
+	ts.Listener.Close()
+	ts.Listener = testListener
+	return ts
+}
```

Comment:
End file with newline.
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline